robot_sf/gym_env/env_util.py

```
@@ -49,28 +48,48 @@ def init_collision_and_sensors(
     # Initialize occupancy objects for each robot for collision detection
-    occupancies = [ContinuousOccupancy(
-        sim.map_def.width, sim.map_def.height,
-        lambda: sim.robot_pos[i], lambda: sim.goal_pos[i],
-        lambda: sim.pysf_sim.env.obstacles_raw[:, :4], lambda: sim.ped_pos,
-        robot_config.radius, sim_config.ped_radius, sim_config.goal_radius)
-        for i in range(num_robots)]
+    occupancies = [
+        ContinuousOccupancy(
+            sim.map_def.width,
+            sim.map_def.height,
+            lambda: sim.robot_pos[i],
```

Ruff (0.8.0)

Function definition does not bind loop variable `i`

(B023)